doc/generic/pgf/text-en/pgfmanual-en-library-decorations.tex

```diff
@@ -1570,6 +1570,66 @@ \subsection{Text Decorations}
     \end{key}
 \end{decoration}
 
+\begin{decoration}{3d text along path}
+    This decoration is almost the same to the |text along path| decoration
+    except that each character is placed on different coordinate plane
+    specified by two unit vectors. The $x$-direction is the tangent direction
+    of the point where the character locates on the path. The $y$-direction is
+    a fixed direction or the direction from the point where the character
+    locates to a fixed point.
+    %
+\begin{codeexample}[preamble={\usetikzlibrary{decorations.text}}]
+\begin{tikzpicture}[decoration={3d text along path,
+  text={|\large\color{red}|3d|| text along path!},
+  text align=center, 3d raise=1cm-.5ex, yvec=60}]
+
+  \draw [fill=gray!30]
+    (0, 2) arc (120:0:2cm) -- ++(60:2cm) arc (0:120:2cm) -- cycle;
+  \path [postaction={decorate}] (0,2) arc (120:0:2cm);
+\end{tikzpicture}
+\end{codeexample}
+\end{decoration}
+
+    \begin{key}{/pgf/decoration/yvec=\marg{number or point} (initially 90)}
+        Set the $y$-direction of every character coordinate plane. If a number
+        is provided, the $y$-direction is the \meta{number}. If a point is
+        provided, the $y$-direction is the direction from the point where the
+        character locates to this \meta{point}.
+        %
+\begin{codeexample}[preamble={\usetikzlibrary{decorations.text}}]
+\begin{tikzpicture}[decoration={3d text along path,
+    text={|\large\color{red}|3d|| text along path!},
+    text align=center, 3d raise=.8ex, yvec={([shift={(60:4cm)}]1.5,1)}}]
+
+  \draw [fill=gray!30]
+    (0, 2) arc (120:0:2cm) -- ([shift={(60:4cm)}]1.5,1) -- cycle;
+  \path [postaction={decorate}] (0,2) arc (120:0:2cm);
+\end{tikzpicture}
+\end{codeexample}
+    \end{key}
+
+    \begin{key}{/pgf/decoration/3d raise=\marg{number or dimension} (initially
+      0pt)}
+        The characters are raised in the $y$-direction of every charater
```

Comment:
```suggestion
        The characters are raised in the $y$-direction of every character
```

Reply:
Thanks for review！But I am afraid that I can't push a new commit until 10 days later... I'll do it as soon as I go home.